src/models.py

```diff
@@ -19,6 +19,10 @@ class Address:
     """
 
     def __init__(self, address: str):
+        # Dune uses \x instead of 0x (i.e. bytea instead of hex string)
+        # This is just a courtesy to query writers,
+        # so they don't have to convert all addresses to hex strings manually
+        address = address.replace("\\x", "0x")
```

Comment:
It would be great, if the class could deal with both cases, and not just one. Meaning if ithe string starts with 0x, let it be, if it starts with \\x, you would change it.

Reply:
This class does deal with both cases. Have you seen the unit tests?

Reply:
I see! the replace will simply not operate on 0xADDRESS at all. Cool